### PR TITLE
fix(quantic): fix broken Cypress test

### PR DESCRIPTION
**Pull request by lbergeron**
https://coveord.atlassian.net/browse/SFINT-4116

An issue was introduced when adding the missing tests. Due to the way our end to end tests are written, we ran into [this pitfall](https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Aliases-are-reset-before-each-test).

In order to quickly fix the issue, I just moved the `Expect.facetValuesEqual()` expectation in its own `describe` to avoid losing the alias. We'll need to discuss about a solution for the longer term.

This PR also bumps the Cypress version as one test was failing on a particular workstation. Upgrading Cypress fixed the issue on that environment.

### CODE REVIEW

**Review by nathanlb — APPROVED** [review]
dope

**Comment by lvu285** [comment]
> https://coveord.atlassian.net/browse/SFINT-4116
> 
> An issue was introduced when adding the missing tests. Due to the way our end to end tests are written, we ran into [this pitfall](https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Aliases-are-reset-before-each-test).
> 
> In order to quickly fix the issue, I just moved the `Expect.facetValuesEqual()` expectation in its own `describe` to avoid losing the alias. We'll need to discuss about a solution for the longer term.
> 
> This PR also bumps the Cypress version as one test was failing on a particular workstation. Upgrading Cypress fixed the issue on that environment.

I don't like alias much, because not all the time we can generate Aliases from the beginning of the test. 
And remember the name of all Aliases/or cross reference from file to file is not friendly for aliases now for me.

**Review by lvu285 — APPROVED** [review]

File: packages/quantic/cypress/integration/facets/facet/facet.cypress.ts [hunk 1]
```diff
@@ -282,43 +292,65 @@ describe('Facet Test Suite', () => {
           });
           cy.wait(InterceptAliases.Search);
           FacetSelectors.showMoreButton().click();
-          aliasFacetValues();
         }
 
         describe('verify rendering', () => {
           before(showMoreValues);
 
-          Expect.facetValuesEqual(indexFacetValuesAlias);
```
[the remark this answers is not in the document]
Comment (nathanlb) [reply packages/quantic/cypress/integration/facets/facet/facet.cypress.ts:291]: I believe it isn't removed but just placed elsewhere, on line 309

**Comment by lbergeron** [comment]
> > https://coveord.atlassian.net/browse/SFINT-4116
> > An issue was introduced when adding the missing tests. Due to the way our end to end tests are written, we ran into [this pitfall](https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Aliases-are-reset-before-each-test).
> > In order to quickly fix the issue, I just moved the `Expect.facetValuesEqual()` expectation in its own `describe` to avoid losing the alias. We'll need to discuss about a solution for the longer term.
> > This PR also bumps the Cypress version as one test was failing on a particular workstation. Upgrading Cypress fixed the issue on that environment.
> 
> I don't like alias much, because not all the time we can generate Aliases from the beginning of the test. And remember the name of all Aliases/or cross reference from file to file is not friendly for aliases now for me.

I understand you prefer to rely on DOM elements to validate expectations. However, when it comes to monitoring requests and responses, relying on aliases is the way to go in Cypress. My understanding of the Cypress documentation is that one should not have to remember a ton of global aliases because aliases should be defined in the scope of a test. What makes it complicated right now is that each expectation is represented as a single test, and Cypress is not meant to be used this way.

In addition, it means that any expectation that rely on an alias must be in its own `describe` block, which also means the page has to be reloaded completely. Actually if we want to validate that a facet renders correctly, has values in the right order, and triggers a UA call, we have to split it over 3 `describe` blocks. We should be able to validate all of this in a single page load.

I fear that we'll constantly be fighting against Cypress if we continue writing tests the way we started.

**Pull request merged**